<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RasterFile</class>
 <widget class="QDialog" name="RasterFile">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>342</width>
    <height>305</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Remove Clouds</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0" colspan="2">
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="image">
         <property name="text">
          <string>Image:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="referenceImage"/>
       </item>
       <item>
        <widget class="QPushButton" name="PushButtonOpenImage">
         <property name="text">
          <string>Open Image...</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="0" colspan="2">
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QLabel" name="shape">
         <property name="text">
          <string>Shape:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="referenceShape"/>
       </item>
       <item>
        <widget class="QPushButton" name="PushButtonOpenShape">
         <property name="text">
          <string>Open Shape...</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="2" column="0">
      <widget class="QListWidget" name="listWidgetTif">
       <property name="dragDropMode">
        <enum>QAbstractItemView::InternalMove</enum>
       </property>
       <property name="defaultDropAction">
        <enum>Qt::TargetMoveAction</enum>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QListWidget" name="listWidgetRaster">
       <property name="dragDropMode">
        <enum>QAbstractItemView::InternalMove</enum>
       </property>
       <property name="defaultDropAction">
        <enum>Qt::TargetMoveAction</enum>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="PushButtonImage">
         <property name="text">
          <string>Image...</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="PushButtonRemoveImage">
         <property name="text">
          <string>Remove</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="3" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QPushButton" name="PushButtonShape">
         <property name="text">
          <string>Shape...</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="PushButtonRemoveShape">
         <property name="text">
          <string>Remove</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="4" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QPushButton" name="PushButtonOK">
         <property name="text">
          <string>OK</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="PushButtonCancel">
         <property name="text">
          <string>Cancel</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
  <zorder></zorder>
  <zorder>listWidgetTif</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>PushButtonOK</zorder>
  <zorder>layoutWidget_2</zorder>
  <zorder>shape</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
